### PR TITLE
Create 0000-01-02-Savidracz-fatalj.md

### PR DESCRIPTION
**Pull request by Savith07**
ok

### CODE REVIEW

**Comment by Savidracz-fatal** [comment]
shit

**Comment by Savidracz-fatal** [comment]
No Description provided

**Comment by Savidracz-fatal** [comment]
no description provided

File: _posts/0000-01-02-Savidracz-fatal.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```
Comment (Savidracz-fatal) [inline _posts/0000-01-02-Savidracz-fatal.md:5]: ok

**Review by Savith07 — COMMENTED** [review]
noice

**Pull request merged**